sherpa-onnx/csrc/stack.cc

```diff
@@ -76,13 +76,13 @@ Ort::Value Stack(OrtAllocator *allocator,
   T *dst = ans.GetTensorMutableData<T>();
 
   for (int32_t i = 0; i != leading_size; ++i) {
-      for (int32_t n = 0; n != static_cast<int32_t>(values.size()); ++n) {
-          const T *src = values[n]->GetTensorData<T>();
-          src += i * trailing_size;
+    for (int32_t n = 0; n != static_cast<int32_t>(values.size()); ++n) {
```

Comment:
I suggest that you handle the case `values.size() == 1` specially.

If the input vector contains only a single tensor, we return a copy of it after changing its shape.

Reply:
@csukuangfj  once things are working, I will add code to handle the `values.size() == 1` case specifically. Thanks.

Reply:
@csukuangfj  I spent some time to think about how to deal with the special `values.size() == 1` case efficiently. `Cat()` can do a simple `Clone()`. But for `Stack()`, I don't see how we can have a shortcut to speed things up. I still need to get `v0_shape` as `Stack()` can happen in `dim=0` or `dim=1`, which result in different outputs. 

Let me know if you have other suggestions. For now, I recommend not to add code to deal with this special case. Thanks.